static/app/utils/replays/hydrateBreadcrumbs.tsx

```diff
@@ -25,6 +25,10 @@ export default function hydrateBreadcrumbs(
             description: t('Encountered an error while hydrating'),
           };
         }
+        // Logcat and Timber are considered a console frame instead of a custom breadcrumb frame
+        if (frame.category === 'Logcat' || frame.category === 'Timber') {
+          frame.category = 'console';
+        }
```

Comment:
I'd be careful of doing this as we would be mutating the original frame... if we wanted to preserve the original frame, we could instead move the logic down to the return value. that way we can avoid any mutations to the original frame object.